pkgs/by-name/c-/c-periphery/package.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
```

Comment:
nit: maybe this should be dedented?

Reply:
hm, what do you mean?

Reply:
Either you already fixed it, GitHub did something weird, or I'm blind. Before your update, it appeared like the entirety of this block was indented with two spaces.